setup.py

```diff
@@ -190,7 +190,7 @@ def run_tests(self):
             'osc2midi = osc2midi.main:main [osc2midi]',
         ]
     },
-    tests_require=['pytest'],
+    tests_require=['pytest','mock'],
```

Comment:
I admit that I've broken a few PEP-8 rules here myself.

Normally in projects I have flake8/pycodestyle intergrated into tox, but unfortunately not in this project, since the main part of the code is not Python but Cython code. I'll have to configure flake8 in `setup.cfg` to check only the correct files. I'll open another ticket for this.